.github/workflows/tests.yml

```diff
@@ -27,10 +27,10 @@ jobs:
     strategy:
       matrix:
         include:
-          - name: Coverage
+          - name: Test (coverage)
             command: coverage run manage.py test && codecov -X gcov
-          - name: Reverse order
-            command: python manage.py test --reverse
+          - name: Test (shuffled)
+            command: python manage.py test --shuffle
```

Comment:
Shuffle has the advantage that it will converge on finding all inter-test dependencies over time. Its randomness has the disadvantage that tests with inter-test dependencies might not be found out in the PR introducing them, but later at some unrelated point in time.

With reverse, we will not find all dependencies (see the ones fixed in this PR), but it will be deterministic: If the PR run passes, all post-merge runs will pass, too.

Opinions?